types/src/json_pretty_printer.rs

```diff
@@ -5,7 +5,25 @@ use alloc::{format, string::String, vec::Vec};
 use serde::Serialize;
 use serde_json::{json, Value};
 
-const MAX_STRING_LEN: usize = 100;
+const MAX_STRING_LEN: usize = 150;
+
+/// Represents the information about a substring found in a string.
+#[derive(Debug)]
+struct SubstringSpec {
+    /// Index of the first character.
+    start_index: usize,
+    /// Length of the substring.
+    length: usize,
+}
+
+impl SubstringSpec {
+    fn new(start_index: usize, length: usize) -> Self {
```

Comment:
The completionist in me wants to add a (admittedly trivial) docstring here. :thinking:

Reply:
Resolved in 34594446fa1a7275bea6322f2283ad9719071a91